ui/src/shared/constants/index.js

```diff
@@ -468,6 +468,8 @@ export const STROKE_WIDTH = {
   light: 1.5,
 }
 
+export const HEARTBEAT_INTERVAL = 10000 // ms
```

Comment:
Could this be computed from the cookie duration? Maybe like 1/2 the cookie duration? I've heard of users setting very low durations (<10s) in EW. Also, if the duration is high we can reduce the request volume on the backend by computing the interval.